components/new-contribution-flow/index.js

```diff
@@ -217,6 +221,32 @@ class ContributionFlow extends React.Component {
     return encodeURIComponent(currentPath);
   }
 
+  /** Validate step profile, create new incognito/org if necessary */
+  /** TODO: create profile for new org */
+  validateStepProfile = async () => {
+    if (!this.state.stepProfile) {
+      return false;
+    }
+
+    // Check if we're creating a new profile
+    if (this.state.stepProfile.id === 'incognito') {
+      this.setState({ submitting: true });
```

Comment:
(we should also fix the other ones below)

```suggestion
      this.setState({ isSubmitting: true });
```